reference-stores/next/pages/pages/[handle].js

```diff
@@ -35,13 +35,13 @@ export async function getStaticPaths() {
 }
 
 export async function getStaticProps({ params }) {
-  const { pages } = await nacelleClient.query({
+  const { data } = await nacelleClient.query({
     query: CONTENT_PAGE_QUERY,
     variables: { handle: `page-${params.handle}` }
   });
   const { page } = await resolvePageData({
     client: nacelleClient,
-    page: pages.edges[0]?.node
+    page: data.pages.edges[0]?.node
```

Comment:
Might be a good opportunity to use optional chaining with `.at`:

```suggestion
    page: data.pages.edges.at(0)?.node
```